Uninstallation.md

```diff
@@ -30,63 +30,94 @@ remove-->
 If you also intend to delete the Chocolatey directory, remove the `-WhatIf`:
```

Comment:
Can we say here where that needs to be removed from?

Reply:
Ah, good catch, absolutely!